daml-assistant/src/DAML/Assistant/Install.hs

```diff
@@ -290,6 +291,9 @@ extractAndInstall env source =
                         liftIO $ setFileMode targetPath (Tar.fileMode info)
                     Tar.FTDirectory -> do
                         liftIO $ createDirectoryIfMissing True targetPath
+                    Tar.FTSymbolicLink bs -> do
+                        liftIO $ createDirectoryIfMissing True parentPath
```

Comment:
I wonder if we should have a check that the symlink is relative.

Reply:
I'll add a check in a separate PR, thanks :)